aries-core/src/stories/navigation.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import { NavigationMenu } from '../js/components';
+
+export const NavigationMenuExample = () => {
+  const items = [
+    { id: 'home', label: 'Home' },
+    { id: 'about', label: 'About' },
+    { id: 'contact', label: 'Contact' },
+  ];
+
+  return <NavigationMenu items={items} />;
```

Comment:
Getting this error:

<img width="641" height="167" alt="Image" src="https://github.com/user-attachments/assets/7758055e-84d2-4c4f-bf78-f73e4e3d3e56" />

I think if you import React at the top of the file tis will go away

Reply:
Added.

Reply:
I don't think we actually need it because of `"jsx": "react-jsx"` option in `tsconfig.json`. I think doing a `cmd+shift+p` in VSCode and `TypeScript: Restart TypeScript server` would eliminate the error. 

But doesn't hurt to have; added for now.

Reply:
Ah okay good to know